source/policies/trustee-ballot-policy.rst

```diff
@@ -0,0 +1,43 @@
+Trustee Election Ballot Policy
+==============================
+
+Section 13.1 of our constitution requires that one-third of the elected trustees
+retire at each Annual General Meeting and that the members have the opportunity to
+fill the resulting vacancies (and others that may exist).
+
+In the event that there are the same number of candidates (or fewer) than vacancies
+available, then the election of those candidates shall be via `acclamation <https://en.wikipedia.org/wiki/Acclamation>`_
+
+If there are more candidates than vacancies, then an election via ballot will be carried
+out using `sequential proportional approval voting <https://en.wikipedia.org/wiki/Sequential_proportional_approval_voting>`_
+
+Election via Acclamation
+------------------------
+- Each candidate is named and presented to the members present at the meeting.
+- For each candidate, the meeting chair asks for a show of hands for those in favour and those against
+  the election of the given candidate.
+- The meeting chair may declare the candidate elected or not or may order a ballot be carried out as they see fit.
+  
+Election via Ballot
+-------------------
+- Each member present is provided with a ballot paper listing the candidates
+- The meeting chair instructs the members that they may cast any number of votes
+  up to the number of vacancies available.
+- The meeting chair informs the members that casting more votes than the number of vacancies
+  renders the ballot paper void.
+- The meeting chair informs the members that casting the same number of votes as the number
+  of vacancies, whilst permissible, is equivalent to casting no vote at all.
+- The meeting chair informs the members that they may only cast a single vote for any
+  given candidate.
```

Comment:
I'm not totally sure how this would come up if the ballot structure is just "place a mark in the candidate's box". What did you have in mind?

Reply:
yeah, probably a little OTT